artemis-server/src/main/java/org/apache/activemq/artemis/core/server/impl/ActiveMQServerImpl.java

```diff
@@ -4689,11 +4689,11 @@ private static void setUnsetQueueParamsToDefaults(QueueConfiguration c) {
 
    private void deployReloadableConfigFromConfiguration() throws Exception {
       if (configurationReloadDeployed.compareAndSet(false, true)) {
-         ActiveMQServerLogger.LOGGER.reloadingConfiguration("security");
+         ActiveMQServerLogger.LOGGER.reloadingConfiguration("security-settings");
```

Comment:
I could see making this "security settings", i.e adding " settings" to make it consistent with the "address settings" one below (which would remain as it was), but I dont think they should strictly match the XML tags since that may in fact not be where the configuration is coming from.